globals/modifiers.css

```diff
@@ -121,3 +147,19 @@
 
 /* Grid helpers */
 .cf {lost-utility: clearfix;}
+
+/* Widths */
+.w1 {  width: 1%;}
```

Comment:
For two items in a table cell layout; if you want an item (like an input) to fill as much space as possible, you must give the other item (like a label) an explicit width. `1%` will make the label as large as it needs to be given it's contents, and the input will fill the rest of the space.